tests/test_bindings.rs

```diff
@@ -114,6 +114,26 @@ fn test_complex() {
     });
 }
 
+#[test]
+#[serial]
+fn test_data() {
+    test_with_leaks!(|| {
+        let mut v = vec![];
+
+        let str: &str = "hello";
+        let bytes: Vec<u8> = str.as_bytes().to_vec();
+        for _ in 0..10_000 {
+            let swift_byte: SRData = SRData::from(&bytes);
```

Comment:
FYI for anyone looking at this PR for an example usage.  This example code is slightly out of date.  See https://github.com/Brendonovich/swift-rs/commit/1506a08b8614f6e8dee9306dc59c08193c9f8a63 for the latest.